### DISCUSSION TITLE
Snapshot 22w11a support

### DISCUSSION THREAD

**Discussion by Greaper88** [Greaper88/magic-sorting-system#1] [General]
Have added the mangrove items to their appropriate wood-related categories.
Added two new groups: Mud and MudWork for the mud related items, and they fallback to the Dirt category.
Added sculker items to the Ocean category, except the sculk sensor,
Sculk Sensor was added to the Redstone category.

Feel free to start a discussion in the discussions tab about relocating the muddy roots: do they belong with wood or mud?

P.S. The "kelp-as-fuel" release directs regular Kelp to the "Raw" group for cooking so the user can make Dried Kelp Blocks which then get routed to the "Coal" group for feeding furnaces. Dried Kelp will go to the regular "Food" group as well.

<hr /><em>This discussion was created from the release <a href='https://github.com/Greaper88/magic-sorting-system/releases/tag/v2.2.0'>Snapshot 22w11a support</a>.</em>

**Comment by mattwillms** [comment]
Thanks for the continued support!

Can you elaborate on how Dried Kelp is routed to food and coal groups?

Perhaps I missed this but is there a way to disable an item frame that would normally be seen as a landing spot? As an example in my base setup the frames can not be seen by the player as they are behind a wall with the hoppers. I'd like to avoid labeling the chests on the player side with signs and would like to use item frames here for labeling without being a teleportation target. Does that make sense?

Muddy Roots question sounds like chicken or the egg 😄

**Reply by Greaper88** [reply]
The vway the kelp is done is as follows: 

"Raw" kelp is added to the "Raw Food" category instead of its usual "Ocean"group

Dried Kelp is part of the "Cooked Food" group. 

And Kelp blocks are part of the "Coal" group.

So you choose the release based on how you want kelp to be treated:  as a normal ocean item, or as a renewable fuel/food item. 


It's not possible (last i knew)  to disable certain item frames (supposedly item frames can't store NBT data). You can always use a similar item though (such as using a spruce log for your displayed item instead of an oak log, etc.), or you can change the item using the config, then running the build script, but that leaves the original issue at play.

**Reply by Greaper88** [reply]
I do have an idea that may work based on another pack i used a while back, but it'll require me digging into things kinda deep, and will take some time...

**Reply by Greaper88** [reply]
I'm also reasonably certain some Nether and Caves and Cliffs items are missing, so some issues being reported on that matter would be great.

**Reply by GedasFX** [reply]
I remember the telescope thingy not working, that was the only problem I remember. Could be that I was using outdated version
